tests/InflectorTest.php

```diff
@@ -232,19 +237,27 @@ public function test_underscore_with_slashes()
 
 	public function test_hyphenate()
 	{
-		$ar = array
-		(
+		$ar = array(
 			'AlterCSSClassNames' => 'alter-css-class-names'
-		)
-
-		+ require __DIR__ . '/cases/underscores_to_dashes.php';
+		);
 
 		$inflector = self::$inflector;
 
 		foreach ($ar as $str => $hyphenated)
 		{
 			$this->assertEquals($hyphenated, $inflector->hyphenate($str));
 		}
+
+		foreach (require __DIR__ . '/cases/underscores_to_dashes.php' as $underscore => $dash)
+		{
+			$this->assertEquals($dash, self::$inflector->hyphenate($underscore));
+		}
+
+		foreach (require __DIR__ . '/cases/misc_to_dashes.php' as $misc => $dash)
```

Comment:
Please rename the file `misc_to_dashes` as `misc_to_hyphens`, since `dasherize()` is only converting `_` to `-`.